TappNetwork/home-visiting-app-web
# fix grid layout on dashboard for web, pointer cursor for sidebar link…

# Details
The resource share modal now has a cancel button, so that it can be exited without sending share on mobile. In the process, I noticed that the web version of the dashboard was rendering incorrectly due to changes made for the mobile builds and fixed it. In addition, I changed the cursor style for the sidebar links to pointer. 

# Steps to Test
1. Go to resource center, click share button on a resource, and confirm cancel button works as expected.
2. Hover mouse over sidebar links to ensure pointer cursor
3. The section description cards in the dashboard should be three to a row and centered in the window.

# Task Linked 
[3kgf5xz](https://app.clickup.com/t/3kgf5xz)

#  Visual Changes
<img width="1500" alt="Screen Shot 2022-09-30 at 10 24 17 AM" src="https://user-images.githubusercontent.com/29612767/193314992-cc575f57-0e35-4d28-b773-3b9e89891f1f.png">
